tests/FakeItEasy.Specs/CreationSpecs.cs

```diff
@@ -123,6 +123,24 @@ public interface ICollectionItem
                 .x(() => fakes.Should().OnlyContain(item => Fake.GetFakeManager(item) != null));
         }
 
+        [Scenario]
+        public void InterfaceWithAlikeGenericMethod()
+        {
+            "Given an interface with methods having multiple generic arguments that can be inheirited"
+                .See<IInterfaceWithSimilarMethods>();
+
+            "It should create fake instance of this"
+                .x(() => this.CreateFake<IInterfaceWithSimilarMethods>());
+

```

Comment:
super nitpick that I wouldn't mention if I wasn't already commenting: we have Style rules that prohibit empty lines at the end of the methods. Could you remove this?